terraform/environments/core-vpc/vpc.tf

```diff
@@ -187,6 +187,31 @@ module "dns_zone_extend" {
   vpc_id      = module.vpc[each.key].vpc_id
   dns_domain  = ".modernisation-platform.internal"
 }
+locals {
+  private-application-zones = {
+      laa = "legalservices.interal",
+      hmpps = "hmpps.internal"
+    
+    }
+}
+data "aws_route53_zone" "private_dns_lookup" {
```

Comment:
This is better done in the module - that way you can supply the name of the zone (eg, `each.value`) and dynamically get the zone_id

Reply:
removed data entry from vpc and placed in module